tools/Azure.Mcp.Tools.Aks/src/Options/AksOptionDefinitions.cs

```diff
@@ -8,10 +8,10 @@ public static class AksOptionDefinitions
     public const string ClusterName = "cluster";
 
     public static readonly Option<string> Cluster = new(
-        $"--{ClusterName}",
-        "AKS Cluster name."
+        $"--{ClusterName}"
     )
     {
-        IsRequired = true
+        Description = "AKS Cluster name.",
```

Comment:
Not your PR, just general comment, I wonder if this description should be changed to

"Azure Kubernetes service (AKS) cluster name."

That way it can key in on both the full name and acronym.